src/Umbraco.Tests.Common/Builders/ContentBuilder.cs

```diff
@@ -48,53 +54,119 @@ public ContentTypeBuilder AddContentType()
             var name = _name ?? Guid.NewGuid().ToString();
-            var creatorId = _creatorId ?? 1;
+            var creatorId = _creatorId ?? 0;
             var level = _level ?? 1;
             var path = _path ?? $"-1,{id}";
             var sortOrder = _sortOrder ?? 0;
             var trashed = _trashed ?? false;
             var culture = _cultureInfo?.Name ?? null;
+            var propertyValues = _propertyValues ?? null;
+            var propertyValuesCulture = _propertyValuesCulture ?? null;
+            var propertyValuesSegment = _propertyValuesSegment ?? null;
 
             if (_contentTypeBuilder is null && _contentType is null)
             {
-                throw new InvalidOperationException("A member cannot be constructed without providing a member type. Use AddContentType() or WithContentType().");
+                throw new InvalidOperationException("A content item cannot be constructed without providing a member type. Use AddContentType() or WithContentType().");
             }
 
             var contentType = _contentType ?? _contentTypeBuilder.Build();
 
-            var content = new Content(name, parentId, contentType, culture)
+            Content content;
+            if (parent != null)
+            {
+                content = new Content(name, parent, contentType, culture)
+                {
+                    ParentId = parent.Id
+                };
+            }
+            else
             {
-                Id = id,
-                Key = key,
-                CreateDate = createDate,
-                UpdateDate = updateDate,
-                CreatorId = creatorId,
-                Level = level,
-                Path = path,
-                SortOrder = sortOrder,
-                Trashed = trashed,
-            };
+                content = new Content(name, parentId, contentType, culture);
+            }
+
+            content.Id = id;
+            content.Key = key;
+            content.CreateDate = createDate;
+            content.UpdateDate = updateDate;
+            content.CreatorId = creatorId;
+            content.Level = level;
+            content.Path = path;
+            content.SortOrder = sortOrder;
+            content.Trashed = trashed;
 
             foreach (var cultureName in _cultureNames)
             {
                 content.SetCultureName(cultureName.Value, cultureName.Key);
             }
 
-
-            if (_propertyDataBuilder != null)
+            if (_propertyDataBuilder != null || propertyValues != null)
             {
-                var propertyData = _propertyDataBuilder.Build();
-                foreach (var kvp in propertyData)
+                if (_propertyDataBuilder != null)
+                {
+                    var propertyData = _propertyDataBuilder.Build();
+                    foreach (var kvp in propertyData)
+                    {
+                        content.SetValue(kvp.Key, kvp.Value);
+                    }
```

Comment:
```suggestion
                    foreach (var keyValuePair in propertyData)
                    {
                        content.SetValue(keyValuePair.Key, keyValuePair.Value);
                    }
```

kvp can be a bit confusing at first requiring you to do a mental double take, keyValuePair is more clear :)